docs/pages/enroll-resources/workload-identity/aws-oidc-federation.mdx

```diff
@@ -0,0 +1,431 @@
+---
+title: Configuring Workload Identity and AWS OIDC Federation with JWTs
```

Comment:
Would it be possible to frame this title more as a high-level objective (e.g., "Authenticate to AWS services with Workload Identity")? Saying "with JWTs" sounds like an implementation detail to me.

Reply:
Unfortunately in this case, we also have a guide for authenticating to AWS services using Workload Identity with certificates.

Reply:
433634cf78